tests/run/i4947c.scala

```diff
@@ -1,4 +1,4 @@
-object Aux {
+object Aux_ {
```

Comment:
I would just rename to something like `Foo`

Reply:
Ok, will rename to Foo and amend the commit